style.css

```diff
@@ -0,0 +1,119 @@
+body {
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+.header {
+  margin-top: 10px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 15px 15px;
+}
+
+.brand {
+  width: 90px;
+  color: #28352f;
+  font-size: 20px;
+  text-align: center;
+  line-height: 28px;
+  height: 28px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+a {
+  text-decoration: none;
+  color: #28352f;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url('header-illsutration-mobile.svg');
+  background-repeat: no-repeat;
+}
+
+.headline-content {
+  padding-left: 20px;
+  width: 327px;
+  font-size: 40px;
+  text-align: center;
+  font-style: normal;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+}
+
+.intro-1 {
+  color: #172b4d;
+  font-size: 38px;
+  font-family: 'Inter', sans-serif;
+}
+
+.intro-2 {
+  color: #36b37e;
+  font-size: 32px;
+  font-family: 'Inter', sans-serif;
+}
+
+.description {
+  width: 327px;
+  height: 164px;
+  padding-left: 24px;
+  font-weight: 300;
+  font-size: 20px;
+  text-align: center;
```

Comment:
[REQUIRED] Please stick with margin and width required in figma desgin
![image](https://user-images.githubusercontent.com/74744787/158792825-6d7a9a29-0b44-4dd5-97bb-c93224bed643.png)
![image](https://user-images.githubusercontent.com/74744787/158792888-85fffd49-7815-4793-8fe8-1707e76d6f0b.png)

Reply:
I have also fixed the margins according to the Figma design
See the screenshot below.
<img width="766" alt="Screenshot 2022-03-17 at 1 21 06 PM" src="https://user-images.githubusercontent.com/30483991/158798994-bbe68225-aabd-4607-8335-5e540dea1b30.png">